femtools/Particles.F90

```diff
@@ -379,14 +422,18 @@ subroutine create_single_particle(detector_list, xfield, position, id, type, nam
     integer :: element
     real, dimension(:,:), intent(in), optional :: attribute_vals
     integer, dimension(3), intent(in) :: attribute_size
+    logical, intent(in), optional :: global
```

Comment:
Might be worth adding a comment that `global=.true.`  requires the call to be collective with the same position asked for on all processes.